### DISCUSSION TITLE
Use of x_cp_bar

### DISCUSSION THREAD

**Discussion by pelljam** [OpenFAST/openfast#979] [General]
Often in the example aerofoil input files, you'll see it documented that x_cp_bar  is ignored if UAMod<>1. I'm not sure that this is true?

Thanks,
James

**Comment by bjonkman** [comment]
@pelljam, it looks like you are correct. `x_cp_bar` is definitely used in the calculation of Cm through the `Cm_v` variable when UAMod = 1, 2, and 3: https://github.com/OpenFAST/openfast/blob/5e25d5d1352e466e78049db122f549003219f34f/modules/aerodyn/src/UnsteadyAero.f90#L3327

@ebranlard, do you know if that is just a typo in the documentation (readthedocs + example input files) or if that is a problem in the code?
